README.md

```diff
@@ -102,23 +144,30 @@ The emulator can also be stopped by closing the command window.
 
 - The configuration of the overall emulator is controlled by the *emulator-config.json* file in the directory where the emulator is installed. (This is the **Swordfish** folder in the installation steps above.) Instructions for using this file can be found in the Redfish Interface Emulator *README.md* file.
 
-- The Swordfish API Emulator code files generally do not overlap the Redfish Interface Emulator code files, but there are currently two files that must contain code for both the Redfish Interface Emulator and the Swordfish API Emulator:
-  - *api_emulator\resource_manager.py*
-  - *api_emulator\utils.py*
+- The Swordfish API Emulator leverages the general structure of the Redfish Interface Emulator and some of the utility functions, but replaces most of the code for the Redfish and Swordfish services and objects.  This service functions in a fully dynamic mode, meaning that objects generally support modification according to the Redfish and Swordfish specifications.
+
+- The Swordfish API Emulator supports the following headers:
+  --  RESP_HEADERS_CONTENT_TYPE
```

Comment:
Not sure if this is intentional, but having the double-dash changes the markdown rendering here. If you keep the indentation, but change it to a single dash, then it would be rendered as sub-bullets in the list.